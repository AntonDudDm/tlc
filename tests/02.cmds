for b in `jot 99`; do
	{ jot -s n $b; echo -n _; } |
	awk -v RS=_ -v ORS=_ '{system("sleep 0.01"); print}'
done
