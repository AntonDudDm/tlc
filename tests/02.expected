[0] %% got            [100] %% got _
